lib/rspec/core/formatters/base_formatter.rb

```diff
@@ -51,6 +51,11 @@ def close(_notification)
           restore_sync_output
         end
 
+        # @private
+        def change_output(output)
```

Comment:
Our convention would just be to create an `attr_writer` or `attr_accessor` for output, it also can't be private as every formatter will have to implement this method.